### PR TITLE
Update configuration.md

### LINKED ISSUE #1999: Describe header forwarding for dashboard when using OIDC with a reverse proxy
## Scenario

- You are running the dashboard behind a reverse proxy.
- You are using OpenIdConnect as authentication.

## Problem

The redirect URI is generate using the http scheme and won't be accepted by Entra ID for example.

## Solution

You need to set the following environment variable to enable forwarded headers, which will enable the dashbaord to generate the correct https scheme redirect URI.

`ASPNETCORE_FORWARDEDHEADERS_ENABLED = true`

## Request for change

Include this in a small section in the docs to show that this environment variable exists, maybe a small hint field in the OpenIdConnect section as well.

### PR DESCRIPTION
**Pull request by denisulmer**
## Summary

Add item to describe ASPNETCORE_FORWARDEDHEADERS_ENABLED. Also add a note in the authentication section.

Fixes #1999

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/fundamentals/dashboard/configuration.md](https://github.com/dotnet/docs-aspire/blob/1ad4554c51afb2637220ec91e69fb9a9ff00970d/docs/fundamentals/dashboard/configuration.md) | [Dashboard configuration](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/configuration?branch=pr-en-us-2000) |

### CODE REVIEW

**Comment by IEvangelist** [comment]
Hi @denisulmer

@JamesNK is most familiar with the .NET Aspire dashboard, let's double check with him on the issue/PR.

File: docs/fundamentals/dashboard/configuration.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Alternatively, these same values could be configured using a JSON configuration
 | `DOTNET_DASHBOARD_UNSECURED_ALLOW_ANONYMOUS` | `false` | Configures the dashboard to not use authentication and accepts anonymous access. This setting is a shortcut to configuring `Dashboard:Frontend:AuthMode` and `Dashboard:Otlp:AuthMode` to `Unsecured`. |
 | `DOTNET_DASHBOARD_CONFIG_FILE_PATH` | `null` | The path for a JSON configuration file. If the dashboard is being run in a Docker container, then this is the path to the configuration file in a mounted volume. This value is optional. |
 | `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` | `null` | The gRPC endpoint to which the dashboard connects for its data. If this value is unspecified, the dashboard shows telemetry data but no resource list or console logs. This setting is a shortcut to `Dashboard:ResourceServiceClient:Url`. |
+| `ASPNETCORE_FORWARDEDHEADERS_ENABLED` | `null` | If set to `true` this will enable header forwarding to the dashboard. This is often needed when running behind a reverse proxy. |
```
Comment (JamesNK) [inline docs/fundamentals/dashboard/configuration.md:75]:
```suggestion
```
This is an example of using a regular ASP.NET Core env var with the dashboard. While it is technically a valid config option, I don't think we should list every ASP.NET Core config option here.

File: docs/fundamentals/dashboard/configuration.md [hunk 2]
```diff
@@ -92,6 +93,9 @@ Browser token authentication works by the frontend asking for a token. The token
 | `Authentication:Schemes:OpenIdConnect:ClientSecret` | `null` | A secret that only the real RP would know. |
 | Other properties of <xref:Microsoft.AspNetCore.Builder.OpenIdConnectOptions> | `null` | Values inside configuration section `Authentication:Schemes:OpenIdConnect:*` are bound to `OpenIdConnectOptions`, such as `Scope`. |
 
+> [!NOTE]
+> In case you are using `OpenIdConnect` as authentication mode behind a reverse-proxy that terminates SSL, check if you need `ASPNETCORE_FORWARDEDHEADERS_ENABLED` to be set.
```
Comment (JamesNK) [inline docs/fundamentals/dashboard/configuration.md:97]:
```suggestion
> Additional configuration may be required when using `OpenIdConnect` as authentication mode behind a reverse-proxy that terminates SSL. Check if you need `ASPNETCORE_FORWARDEDHEADERS_ENABLED` to be set to `true`.
>
> For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](/aspnet/core/host-and-deploy/proxy-load-balancer).
```

**Review by IEvangelist — APPROVED** [review]

**Comment by denisulmer** [comment]
Yeah you are right. Thanks for updating and merging, didn't have time yet. Cheers!

**Pull request merged**